data.sql

```diff
@@ -38,3 +38,11 @@ WHERE Name = 'Charmander' or Name = 'Squirtle' or Name = 'Blossom';
 UPDATE animals
 SET owner_id = 5
 WHERE Name = 'Angemon' or Name = 'Boarmon';
+
+INSERT INTO vets (name,age,date_of_graduation) VALUES('Willian Tatcher', 45, (to_date('04/23/2000','mm/dd/yyyy')) ),('Maisy Smith', 26, (to_date('01/17/2019','mm/dd/yyyy')) ),('Stephanie Mendez', 64, (to_date('05/04/1981','mm/dd/yyyy')) ),('Jack Harkness', 38, (to_date('06/08/2008','mm/dd/yyyy')) );
```

Comment:
- [ ] Let us use this way to insert the data into vets table to keep our code clean and clear as well as easy to read.

  `INSERT INTO vets (name, age, date_of_graduation)
   VALUES ('William Tatcher', 45, '2000-04-23');`

  ` INSERT INTO vets (name, age, date_of_graduation)
   VALUES ('Maisy Smith', 26, '2019-01-17');`

  ` INSERT INTO vets (name, age, date_of_graduation)
   VALUES ('Stephanie Mendez', 64, '2019-05-17');`

   `INSERT INTO vets (name, age, date_of_graduation)
   VALUES ('Jack Harknes', 38, '2008-06-08'); `

Reply:
Thanks for the alternative. Done